src/mudlet-lua/lua/GUIUtils.lua

```diff
@@ -380,6 +380,23 @@ function disableGaugeClickthrough(gaugeName)
   disableClickthrough(gaugeName .. "_text")
 end
 
+--- Set gauge to have a tooltip
+--- @param gaugeName
+--- @param text the tooltip text
+--- @param duration tooltip duration
+function setGaugeToolTip(gaugeName, text, duration)
+  duration = duration or 0
+  assert(gaugesTable[gaugeName], "setGaugeToolTip: no such gauge exists.")
```

Comment:
Not something to fix here, but I realised we have an inconsistency here between Geyser and the C++ API. One does an error but the other a nil+msg.